server/middlewares/userValidator.js

```diff
@@ -0,0 +1,138 @@
+import { check } from 'express-validator/check';
+import { user } from '../database/models';
+import { capitaliseFirstLetter, makeLowerCase } from '../helpers';
```

Comment:
The capitaliseFirstLetter and makeLowerCase cannot be deconstructed this way, it indicates not found

Reply:
Why? I think that's valid. Secondly, it's not showing any errors on my end. Have you tried refreshing your VS Code?